src/pages/Admin/Venue/TemplateCard/TemplateCard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
```

Comment:
Extracted `TemplateCard` in own separate dir/files for ease of maintenance and future reuse (copy from).